packages/tailwindcss/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@formkit/tailwindcss",
+  "repository": "https://github.com/formkit/formkit",
+  "version": "1.0.0-beta.4",
+  "description": "A FormKit plugin for Tailwind that supplies custom variants based on input state",
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "contributors": [
+    "Andrew Boyd <andrew@formkit.com>"
+  ],
+  "license": "MIT",
+  "type": "module",
+  "devDependencies": {
+    "@types/tailwindcss": "^3.0.9",
```

Comment:
I think tailwind here is more of a `peerDependency` than a dev dependency since you are probably adding formkit to your tailwind and not the other way around.

Also, unless you know that `3.0.9` and `3.0.23` is when they introduces some specific feature set we are using we probably want this to get knocked as far back as possible — likely `^3.0.0`.